ToyBox/classes/MainUI/ActionButtons.cs

```diff
@@ -48,9 +48,14 @@ public static class ActionButtons {
                 UI.Space(width + 3);
             }
         }
-        public static void BlueprintActionButton(this BlueprintAction action, UnitEntityData unit, SimpleBlueprint bp, Action buttonAction, float width) {
+        public static void BlueprintActionButton(this BlueprintAction action, UnitEntityData unit, SimpleBlueprint bp, Action buttonAction, float width, bool shouldLocalize = false) {
             if (action != null && action.canPerform(bp, unit)) {
-                UI.ActionButton(action.name, buttonAction, width == 0 ? UI.AutoWidth() : UI.Width(width));
+                if (shouldLocalize) {
```

Comment:
The BlueprintAction should be constructed with a localized name. Please do that and remove this flag